src/components/Login/Login.js

```diff
@@ -1,54 +1,52 @@
 import React, { Component } from 'react';
 import './Login.scss';
+import { Redirect } from 'react-router-dom';
 import { connect } from 'react-redux';
-import { setUser } from '../../actions/actions'
+import { setUser } from '../../actions/actions';
+import { getUser } from '../../apiCalls';
+
 
 class Login extends Component {
   constructor() {
     super();
     this.state = {
-      name: '',
       password: '',
       email: '',
-      nameError: '',
       emailError: '',
-      passwordError: ''
+      passwordError: '',
+      error: ''
     };
   }
 
   handleChange = event => {
     this.setState({ [event.target.name]: event.target.value });
   };
 
-  verifyInputs = (event) => {
-    event.preventDefault()
-    if(this.state.name !== 'Greg') {
-      this.setState({nameError: 'Incorrect name entered.'})
-    } if(this.state.email !== 'greg@turing.io') {
-      this.setState({emailError: 'Incorrect email entered.'})
-    } if(this.state.password !== 'abc123') {
-      this.setState({passwordError: 'Incorrect password entered.'})
-    } else {
-      this.createUser()
-    }
+  handleSubmit = async (e) => {
```

Comment:
We might not need this verification if the fetch call returns an error message for an incorrect login, this can probably be refactored

Reply:
Most likely it will need to be refactored.